src/main/java/com/example/gsheets/client/GoogleCredentialAPI.java

```diff
@@ -0,0 +1,261 @@
+package com.example.gsheets.client;
+
+import com.example.gsheets.service.domain.Student;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.SheetsScopes;
+import com.google.api.services.sheets.v4.model.*;
+
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.util.*;
+import java.util.stream.IntStream;
+
+import static com.example.gsheets.client.mapper.ClientDTOMapper.*;
+import static com.example.gsheets.service.mapper.StudentMapper.fromDTOClientToStudent;
+
+public class GoogleCredentialAPI {
+    private static final String APPLICATION_NAME = "Google Sheets API Java Quickstart";
+    final static String spreadId = "1ojNfksTD6GzD6pA_rOEFVex0C2kZqn0GdVY3ltnUFhw";
+
+    private static final JsonFactory JSON_FACTORY = GsonFactory.getDefaultInstance();
+    private static final List<String> SCOPES =
+            Collections.singletonList(SheetsScopes.SPREADSHEETS);
+    private static final String KEY_FILE_PATH = "/creds.json";
+
+    public static Credential getCredentials() throws IOException, GeneralSecurityException {
+        GoogleCredential credential = GoogleCredential
+                .fromStream(Objects.requireNonNull(GoogleCredentialAPI.class.getResourceAsStream(KEY_FILE_PATH)))
+                .createScoped(SCOPES)
+                .createDelegated("myname@sheetapi-405408.iam.gserviceaccount.com");
```

Comment:
what is it?

Reply:
It's create google credential service account for server-to-server authorization

Reply:
e-mail service account i move to application properties.